### PR TITLE
Added the Festivals category, in alphabetical order. 

### PR DESCRIPTION
**Pull request by PravunathSingh**
Here I have tried to add at least one major festival for each letter.

### CODE REVIEW

**Review by gabrielbarker — CHANGES_REQUESTED** [review]
One problem is that values need to be lower case and single words. I've checked and you have about 10 here that are just one word which would be great! Could throw in 'easter' and 'hanukkah' too? Maybe just 'valentines' as well? Let me know your thoughts

You also need to add a test class and add this class to the map in 'pattern.py'. There are more details in contributing.md and in the issue description 🙂

**Comment by PravunathSingh** [comment]
Thanks a lot @gabrielbarker  for the insights, I'll definitely work on the changes and let you know

**Comment by PravunathSingh** [comment]
Hey @gabrielbarker I have made the changes, but two of my codecov checks are failing, could you provide some insights regarding them

**Comment by GoldDust69** [comment]
> Hey @gabrielbarker I have made the changes, but two of my codecov checks are failing, could you provide some insights regarding them

Hey there, it looks like you're still missing the test class for this which needs to be created in test/categories :)

**Comment by GoldDust69** [comment]
Hey there, it looks like you're still missing the test class for this which needs to be created in test/categories :)

**Comment by PravunathSingh** [comment]
Hey, @GoldDust69 thanks for the help :).

**Comment by PravunathSingh** [comment]
Hey, @gabrielbarker I've made all the changes, hopefully this time my changes would be approved. It would be great if you could review them

File: qual_id/categories/anime.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from ..category import Category
```
Comment (gabrielbarker) [inline qual_id/categories/anime.py:1]: Please can you remove this category file and add it (along with tests etc.) in a separate PR?

**Review by gabrielbarker — CHANGES_REQUESTED** [review]

**Comment by PravunathSingh** [comment]
Hey, @gabrielbarker I've deleted the anime.py file from this PR, as you said. I'm really for that mistake, I didn't notice it was there. :(

**Review by gabrielbarker — APPROVED** [review]
Looks great this time!

**Comment by gabrielbarker** [comment]
Thanks so much for the contribution @PravunathSingh ! 🙂 🎉

**Comment by PravunathSingh** [comment]
Thanks a lot @gabrielbarker for all the suggestions and guidance!!. Learnt a lot while contributing to this.

Looking forward to making more and better contributions to this repo :)

**Comment by gabrielbarker** [comment]
I'm so pleased! Thanks @PravunathSingh !

**Pull request merged**